nav2_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,22 @@
+# Type definitions
+uint8 TYPE_FILE=0
+uint8 TYPE_URL=1
```

Comment:
Do we support URLs? If so, I didn't know that and that's awesome

Reply:
Not yet, but I made the service capable of containing one. We'd just have to modify the map_server code to support it. For now I return an error if it's not set to TYPE_FILE